wiki/Glossary/Grid_Snapping/en.md

```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - lock
+---
+
+# Grid Snapping
+
+**Grid snapping** refers to a button on the right-hand side of the [beatmap editor](/wiki/Beatmap_Editor). Enabling it will force every [hit object](/wiki/Hit_object/) that is interacted with to snap along the intersections of the grid.
```

Comment:
```suggestion
**Grid snapping** refers to the act of using the `Grid Snap` button functionality in the [beatmap editor](/wiki/Beatmap_Editor). Enabling it will force every [hit object](/wiki/Hit_object/) that is interacted with by the creator to move to the nearest position along the intersections of the grid, or, in other words, "snapping." The `Grid Snap` button can be found on the right-hand side of the editor.

<!-- image of Grid Snap button location or the just the Grid Snap button itself -->
```